Lib/dis.py

```diff
@@ -460,7 +460,10 @@ def print_instruction(self, instr, mark_as_current=False):
                     # Only show the fancy argrepr for a CACHE instruction when it's
                     # the first entry for a particular cache value:
                     if i == 0:
-                        argrepr = f"{name}: {int.from_bytes(data, sys.byteorder)}"
+                        data_int = int.from_bytes(data, sys.byteorder)
+                        argrepr = f"{name}: {data_int}"
+                        if name == "counter":
+                            argrepr += f" (value: {data_int >> 4}, backoff: {data_int & 15})"
```

Comment:
I think this should be extracted into a helper function in case we want to add more information later.